config/vufind/combined.ini

```diff
@@ -10,6 +10,8 @@
 ; sublabel = Text to display below the header (optional)
 ; more_link = The text of the link to access more results; set to false to
 ;        disable all "more results" links for this entry.
+; advanced_link = The text of the link to load the advanced search page for
```

Comment:
Why does the link text need to be configurable?

Reply:
The more_link is configurable, so I imagine this is for consistency.

Reply:
Yes consistency was the intent.  Plus there has to be a way to enable/disable it, so there needs to be something in the .ini anyway.

Reply:
> Okay. why is the more_link configurable? :)

That one for @demiankatz  :)

Reply:
I can accept it for consistency. The comments could reflect the fact that they can be translation keys as well, so that anyone doesn't think that it can't be translated.

Reply:
The more_link is configurable because each column of the combined search might need a different custom message (i.e. "Show more results for Solr", "Show more results for Summon", etc.). It's possible that the advanced search is less likely to need that kind of individualization, but I don't think the flexibility hurts.

Reply:
> The comments could reflect the fact that they can be translation keys as well, so that anyone doesn't think that it can't be translated.

Good idea, I will add that to combined.ini.